portal/src/components/cards/BinderHubCard.component.vue

```diff
@@ -0,0 +1,108 @@
+<template>
+  <el-card :body-style="{ padding: '0px' }" class="mx-10 p-5">
+    <h5 class="text-2xl font-medium">BinderHub Filter</h5>
+    <hr class="divider divider-gray pt-2" />
+    <div v-for="(item, index) in binderhubs" :key="index" class="item">
+      <el-row justify="space-between" align="middle">
+        <el-col :span="13">
+          <br>
+          <el-tooltip :content="item.description" placement="left">
+            <el-text>{{ item.name }}</el-text>
+          </el-tooltip>
+        </el-col>
+        <el-col :span="9.5">
+          <br>
+          <a :href="generateUrl(item.url)" target="_blank" rel="noopener noreferrer">
+            <manku-icon name="binderLink" height="25" fill="blue" />
+          </a>
+        </el-col>
+        <el-col :span="1" fixed="right">
+          <br>
+          <el-tooltip :content="authenticationTooltip(item.authentication)" placement="right">
+            <font-awesome-icon :icon="trafficIcon(item.authentication)" size="lg"
+              :style="trafficStyle(item.authentication)" />
+          </el-tooltip>
+        </el-col>
+      </el-row>
+    </div>
+  </el-card>
+</template>
+
+<script>
+
+import binderhubconfiguration from '@/binderhubconfiguration.json';
+import { first } from "lodash";
+
+export default {
+  props: ['gitOrg', 'gitName', 'gitBranch', 'filepath'],
+  mounted() {
+    this.registryBinderhubs();
+    this.notebookLink();
+    this.generateUrl();
+    this.authenticationTooltip();
+    // this.memoryTooltip();
+    this.trafficIcon();
+    this.trafficStyle()
+  },
+  data() {
+    return {
+      binderhubs: [],
+    };
+  },
+
+  methods: {
+    async registryBinderhubs() {
+      const githubToken = binderhubconfiguration.githubToken;
+      if (!githubToken) {
+        console.error('GitHub token is missing.');
+        return;
+      }
+      let registryJson =
+        "https://api.github.com/repos/Language-Research-Technology/BinderHub-registry/contents/BinderHub-registry.json";
```

Comment:
Can we have this not hard coded and added to the configuration.json please

Reply:
Updated now